### ISSUE TITLE
duplicated HTML anchors on page for git diff

### ISSUE THREAD

**Issue by mast-eu** [git/git-scm.com#1331]
The page https://git-scm.com/docs/git-diff contains two options

- `--color-moved[=<mode>]` ([link](https://git-scm.com/docs/git-diff#Documentation/git-diff.txt---color-movedltmodegt)) and
- `--color-moved-ws=<modes>` ([link](https://git-scm.com/docs/git-diff#Documentation/git-diff.txt---color-moved-wsltmodesgt)), 

which have both a parameter `no`.

The HTML anchor for both parameters is identical: https://git-scm.com/docs/git-diff#Documentation/git-diff.txt-no
The link leads to `--color-moved=no`. It is impossible to link directly to `--color-moved-ws=no`, because this is the second occurrence of the anchor within the same page.

#987 describes the same issue on another page.

**Comment by peff** [comment]
Yes as #987 hints at, I don't think we can fix this without rewriting how the anchors are added, because the current regex doesn't know about the hierarchy of items. I think we'd need to integrate more with asciidoc's view of the document, like the suggestion from https://github.com/asciidoctor/asciidoctor/issues/2717.
